tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -5451,5 +5451,44 @@ def test_forward_unique_with_counts():
             _test_unique_with_counts(20, dtype, is_dyn)
 
 
+#######################################################################
+# check graph ir for nn.moments
+# ------------
+
+SEMVER = '#[version = "0.0.5"]\n'
+
+
+def run_from_tensorflow(graph):
```

Comment:
This method is not quite useful though since you only use it once and it only has two lines